calicoctl/resourcemgr/globalnetworkset.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2018 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resourcemgr
+
+import (
+	"context"
+
+	api "github.com/projectcalico/libcalico-go/lib/apis/v3"
+	client "github.com/projectcalico/libcalico-go/lib/clientv3"
+	"github.com/projectcalico/libcalico-go/lib/options"
+)
+
+func init() {
+	registerResource(
+		api.NewGlobalNetworkSet(),
+		api.NewGlobalNetworkSetList(),
+		false,
+		[]string{"globalnetworkset", "globalnetworksets"},
+		[]string{"NAME"},
+		[]string{"NAME", "NETS"},
```

Comment:
Hm, do we have any way to limit the length of what gets put in the NETS column? It could potentially get really really big.

Reply:
Good point; I've seen kubectl truncate with ellipsis in that sort of situation.  I'll look into it